src/main/java/com/google/graphgeckos/dashboard/storage/DatastoreRepository.java

```diff
@@ -119,11 +134,9 @@ public boolean deleteRevisionEntry(String commitHash) throws IllegalArgumentExce
 
         return false;
       }
-
-      return true;
     }
 
-    return false;
+    return true;
```

Comment:
this update doesn't belong to this PR. Could you add it as a follow-up PR, if it's intended?

Reply:
it is intented. I will remove it and create a separate PR for it.